addons/account/models/account_move.py

```diff
@@ -2175,6 +2175,15 @@ def update_containers():
     # LOW-LEVEL METHODS
     # -------------------------------------------------------------------------
 
+    def check_field_access_rights(self, operation, field_names):
+        # TODO in master: update parent method to check `exportable` attribute
+        # of the hacked fields
```

Comment:
@rco-odoo Is this something you would agree to or should we remove the TODO?
I'd like to manage it with the ORM in master, as you probably already guessed.